buildspec_deploy.yml

```diff
@@ -5,6 +5,8 @@ phases:
     commands:
       - echo Installing grow
       - pip install grow pyyaml bibtexparser
+      - echo Installing tailwind
+      - npm install
```

Comment:
i am not actually sure if this will be needed.  grow install may be all that is needed (works on netlify).  on aws codebuild though it may not be as "smart" so npm install should probably be left in